### PR TITLE
chore(objects): Resolve namespace warnings

### PR DESCRIPTION
**Pull request by clairekuang**
Adjusts folder structure for all namespace warnings in `Objects`. Organization is based on namespaces - no namespaces were adjusted.

### CODE REVIEW

**Review by JR-Morgan — APPROVED** [review]

File: Objects/Objects/BuiltElements/Revit/RevitRoof.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
+using System.Collections.Generic;
+using Objects.Geometry;
+using Objects.Utils;
+using Speckle.Core.Kits;
+using Speckle.Core.Models;
+
+namespace Objects.BuiltElements.Revit.RevitRoof;
```
Comment (JR-Morgan) [inline Objects/Objects/BuiltElements/Revit/RevitRoof.cs:7]: I see this one still generates a warning... But this namespace is questionable at best.
Perhaps we can use our "deprecated" namespacing feature of the serializer to deprecate + move these incorrectly namespace classes.

Perhaps to be done in another PR

Comment (clairekuang) [reply Objects/Objects/BuiltElements/Revit/RevitRoof.cs]: Hmm for now I've created a `RevitRoof` folder for this file, but it does seem like the namespace is incorrect. We can fix these in another ticket
